client/src/App.css

```diff
@@ -0,0 +1,10 @@
+* {
+    margin: 0;
+    padding: 0;
+    outline: none;
+}
+
+a {
```

Comment:
don't use tag name for a style.

Reply:
The style for the a tag is a general one such as the *.
It's not customized for any page, it just removes the decoration and fixes the color to white.

Reply:
I think there will be repeated style code for all the (a/Link) tags we use such as the color and the decoration.

Reply:
@aminalakhsham  
i'm with ahmed in this point